superset/assets/src/dashboard/stylesheets/dashboard.less

```diff
@@ -56,7 +56,7 @@ body {
   }
 
   .divider {
-    margin: 5px 0;
+    margin: 5px -10px;
```

Comment:
Can you add before / after pictures of this? I feel like negative margins are usually indicative of larger problems 🤔